make_benchmark_data/process_benchmark_quotes_corenlp_output.R

```diff
@@ -0,0 +1,110 @@
+require(jsonlite)
+require(data.table)
+require(here)
+
+proj_dir = here()
+source(paste(proj_dir, "/utils/scraper_processing_utils.R", sep=""))
+
+
+format_gender_canonical_speaker <- function(in_df){
+
+
+    # if the gender is unknown, but there is a 
+    # gender guess in canonical speaker, take it
+
+    male_pronouns = na.omit(pronouns_df$pronouns[pronouns_df$gender == "MALE"])
+    female_pronouns = na.omit(pronouns_df$pronouns[pronouns_df$gender == "FEMALE"])
+
+    male_idx = which((in_df$gender == "UNKNOWN" | is.na(in_df$gender)) &
+                        tolower(in_df$canonical_speaker) %in% male_pronouns)
```

Comment:
```suggestion
    male_idx = which(
    (in_df$gender == "UNKNOWN" | is.na(in_df$gender)) &
    tolower(in_df$canonical_speaker) %in% male_pronouns
    )
```
Would break down these long lines into multiple lines as above. More whitespace results in easy to read code.